README.md

```diff
@@ -1,70 +1,113 @@
+~~~~
+set HADOOP_HOME=C:\Users\myusername\Documents\cloud-data-access-client-demo
+~~~~
 
-Run `./gradlew build`
+For more info, see:
+-  [Page at cwiki.apache.org](https://cwiki.apache.org/confluence/display/HADOOP2/WindowsProblems)
+- [Page at answers.microsoft.com](https://answers.microsoft.com/en-us/insider/forum/insider_wintp-insider_repair/how-do-i-fix-this-error-msvcp100dll-is-missing/c167d686-044e-44ab-8e8f-968fac9525c5?auth=1)
+</details>
 
 - - - 
-## Usage
+## Run the CDA Client
+<details>
+<summary>Click to expand</summary>
 
-Run the utility by executing the jar from the command line. 
+1. **Configure S3 authentication**: The utility requires AWS credentials to access the source S3 bucket that contains the table data (and optionally to write to a destination S3 bucket). These must be exported in the command line before running the program. For example, use this command after replacing `<secret key>`, `<access key>`, and `<region name>` with values from a user's credentials file located in `~/.aws/credentials`:
 ~~~~
-java -jar cloud-data-access-client-1.0.jar --configPath "config.yaml"
+export AWS_SECRET_ACCESS_KEY=<secret key> AWS_ACCESS_KEY_ID=<access key> AWS_REGION=<region name>
 ~~~~
 
-Where the option --configPath or -c designates the path to the configuration file. See below for an example usage of a configuration file. The utility also relies on AWS credentials that need to be defined through environment variables before running the utility. See below for details about setting AWS credentials.
+   - If you are using awscli with a credentials file and profiles, you will need to use the environment var `AWS_PROFILE`, instead of setting the keys directly. For example:  
+~~~~
+export AWS_PROFILE=myProfile
+~~~~
+* More information can be found here for setting up [AWS Credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
 
-If you are running on a Windows machine, there is an additional utility that must be installed in order for the client to work. See the section below.
+2. Download the sample configuration file from the Git repository folder `/src/test/resources/sample_config.yaml` and save under a new name such as `config.yaml`.
+3. Configure the `config.yaml` file.
+4. Run the utility by executing the jar from the command line with one of these commands:
+  <ul><li>If you are running the CDA Client for the first time (without a <tt>savepoints.json</tt> file from a previous run) or have a large amount of data in the S3 bucket, both reading and writing can take a substantial amount of time depending on your machine. By default, the Java runtime environment [allocates a maximum of 1/4 of the computer's memory](https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gc-ergonomics.html). It may be necessary to increase the memory available to the application for larger amounts of data. For example, run the client with an increased maximum memory allocation of 8 GB ("8g") with this command:
```

Comment:
```suggestion
  <ul><li>If you are running the CDA Client for the first time (without a `savepoints.json` file from a previous run) or have a large amount of data in the S3 bucket, both reading and writing can take a substantial amount of time depending on your machine. By default, the Java runtime environment [allocates a maximum of 1/4 of the computer's memory](https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gc-ergonomics.html). It may be necessary to increase the memory available to the application for larger amounts of data. For example, run the client with an increased maximum memory allocation of 8 GB ("8g") with this command:
```